src/ubuntu/22.04/cross/arm64/Dockerfile

```diff
@@ -4,6 +4,7 @@ FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-crossdeps-local
 RUN apt-get update \
     && apt-get install -y \
         binutils-aarch64-linux-gnu \
+        nodejs \
```

Comment:
@wtgodbe is `nodejs` being pulled in from the Ubuntu archive? If yes, the latest version available in Ubuntu 22.04 is [Node 12](https://launchpad.net/ubuntu/jammy/+source/nodejs).

Reply:
I would image so - sorry, I don't have the most experience on configuring docker containers. What would be the right way to get node 20 installed?

Reply:
I imagine you would need to [install the binary directly](https://github.com/nodejs/help/wiki/Installation) or use the [package manager instructions provided by NodeSource](https://github.com/nodesource/distributions?tab=readme-ov-file#debian-and-ubuntu-based-distributions), although I don't like `curl | bash` they suggest 😕

@mthalman do we have a precedent for situations like this?

Reply:
> @mthalman do we have a precedent for situations like this?

Yes: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/f22565c135a08c02c30fb81f453d9952ab5dc572/src/ubuntu/18.04/helix/webassembly/Dockerfile#L32-L35